cdap-common/src/main/java/co/cask/cdap/common/conf/CConfiguration.java

```diff
@@ -34,7 +34,7 @@
   private static final Logger LOG =
     LoggerFactory.getLogger(CConfiguration.class);
 
-  private CConfiguration() {
+  public CConfiguration() {
```

Comment:
why the change? And it's different than what the comment inside this constructor.